examples/src/main/java/com/mapbox/navigation/examples/core/MultipleStopsActivity.kt

```diff
@@ -188,6 +205,7 @@ class MultipleStopsActivity : AppCompatActivity(), OnMapReadyCallback {
         super.onDestroy()
         mapboxNavigation?.stopTripSession()
         mapboxNavigation?.onDestroy()
+        mapboxNavigation?.removeArrivalController()
```

Comment:
See comment above re: unregister `arrivalObserver` when in `onDestroy`

Reply:
No needed? It's being removed implicitly from `mapboxNavigation?.onDestroy()` https://github.com/mapbox/mapbox-navigation-android/pull/2787#discussion_r415957666